### DISCUSSION TITLE
Review on the intro lesson

### DISCUSSION THREAD

**Discussion by nasirdeenmoh** [skills/.github#782] [Introduction to GitHub]
It's a very intuitive and motivating lesson, I really loved it

**Comment by mahyamadelta** [comment]
how do I start?

**Reply by nasirdeenmoh** [reply]
Access this link:
https://github.com/skills/introduction-to-github
Scroll down to the README section and then click "Start course", then you follow the instructions

**Reply by mahyamadelta** [reply]
thanks
